components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -309,6 +319,7 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
                 prebuildTasks,
                 parentPrebuildTasks,
             });
+            log.info("Disqualified: prebuild tasks have changed");
```

Comment:
This log entries I think should either a) go away entirely or b) be `.debug` if we really need them.

Reply:
Thanks! Only added for debugging when I wasn't getting any base -- that's no longer necessary so removed. ✅